updater/nd_commander_tips/translations/nd_commander_tips.phrases.txt

```diff
@@ -69,6 +69,69 @@
 		"en"		"Placing Gates (Spawns) on the front lines allows your troops to get there faster"
 	}
 	
+	"Limit Classes"
+	{
+		// 11 - To remind the commander that they can limit the classes in their army
+		"en"		"You can use commands like '!maxsnipers 1', in the chat box, to limit the number of concurrent players in each class"
+	}
+	
+	"Slay Bots"
+	{
+		// 12 - To remind the commander that they can reset all the bots with !slaybots
+		"en"		"You can reset all the Bots on your team, by killing them all with the '!slaybots' command, in the chat box"
+	}
+	
+	"Stealth Attacks"
+	{
+		// 13 - To alert them to the potential for stealth attacks
+		"en"		"Keep an eye out for enemy troops, especialy stealths, that attack you in the chair"
+	}
+	
+	"Mine the Bunker"
+	{
+		// 14 - To remind them they can spawn as Support Engie and place daisy cutters to protect them in the bunker	
+		"en"		"You can always place Mines (Daisy Cutters) on the entrance to your bunker, to kill or damage enemies who try and enter"
+	}
+	
+	"Low ammo"
+	{
+		// 15 - To remind the commander that Supply Stations are important to keep troops shootin. 
+		// Could this be triggered like the low average health warnings for Medics ? (would need re-phrasing) 
+		"en"		"Place Supply Stations around your front lines, so as to keep your troops shooting and healthy"
```

Comment:
Place a supply station in base beside the spawn, so troops can teleport back and resupply.

- They shouldn't be playing too many in the front lines because they are expensive and easy to kill.

Reply:
Would this be one to add to the Commander Checklist ?